hibernate-reactive/src/test/groovy/io/micronaut/configuration/hibernate/reactive/multiple/MultipleDataSourceJpaSetupSpec.groovy

```diff
@@ -45,6 +45,7 @@ class MultipleDataSourceJpaSetupSpec extends Specification implements TestProper
         [
                 'datasources.default.url'                           : 'jdbc:h2:mem:mydb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE',
                 'jpa.default.packages-to-scan'                      : 'io.micronaut.configuration.hibernate.reactive.multiple.xyz',
+                'datasources.other.name'                             :  'other',
```

Comment:
This might not be needed with fixes in core 4.6.1, also maybe in [CurrentSessionWithMultipleDataSourcesSpec.groovy](https://github.com/micronaut-projects/micronaut-sql/pull/1403/files#diff-d0309d06a64f076d2cf676469560a631130e30453fe668b5b6d88e24c300ca78)

Reply:
I have reverted it.